docs/Planned-Features.md

```diff
@@ -0,0 +1,29 @@
+# Planned Features
```

Comment:
I guess we should drop this one:

- dynamic_join and date/time types are available.
- I don't have plans to support conditional columns in sqlpp11

Reply:
Maybe we could put the descriptions in `Planned-Features.md` and `New-Features.md` in new or existing pages. For example `Union` could be added to `select.md` and `Custom Query` should probably have its own page. Not sure how to apply that to planned features though, since the description seems uncertain.. might be best to just drop that one completely and rewrite the feature descriptions later.

What do you think?

Reply:
Yes, I suggest dropping `Planned-Features` completely.

And I agree, the content of `New-Fetaures` should be distributed accordingly and the page dropped as well.

But we can easily do that in another PR.